packages/pds/src/api/com/atproto/repo/applyWrites.ts

```diff
@@ -84,6 +87,16 @@ export default function (server: Server, ctx: AppContext) {
         }
       }
 
+      const account = await ctx.authVerifier.findAccount(repo, {
+        checkDeactivated: true,
+        checkTakedown: true,
+      })
```

Comment:
Moving this code down here now performs the authorization checks after status verification, where these usually occur in the opposite order.  I think for consistency we should probably switch it back around.

Reply:
Just looking ahead, I believe there's a similar case with the other endpoints that were updated.  For example, if I am takendown I should fail to authenticate and my write permissions should not be checked.